pkg/fs/local_file_system_windows.go

```diff
@@ -66,4 +66,4 @@ func syncFile(_ *os.File) error {
 
 func CompareINode(srcPath, destPath string) error {
 	return nil
-}
\ No newline at end of file
+}
```

Comment:
This is an unexpected change. Nothing changed.

Reply:
Please don't change it.